### PR TITLE
Chapters can customize email response to applicants

### PR DESCRIPTION
**Pull request by arsduo**
Issue #156 pointed out that it would be useful to let individual chapters provide their own text for the email that gets sent to users after applying. That way, chapters can include their own contact info, etc.

This PR adds that feature. Deans and other chapter admins can add text in the chapter view. When an applicant applies, the main paragraph (between `Hi #{project_name}!` and the application details) will be replaced by that value. 

The email template isn't currently localized, so this PR doesn't worry about localization. Future localization of other strings in the email template shouldn't affect this. (If chapters handle multiple languages, I'd propose it be up to them how to organize content in multiple languages.)

I also noticed that some extra colons would show up in the email because blank answers to extra questions get saved as `""`; this PR checks `.present?` rather than non-falsy to avoid that.

### CODE REVIEW

**Comment by arsduo** [comment]
_You can add text in the chapter edit view:_

screen shot 2017-07-06 at 18 55 24

screen shot 2017-07-06 at 19 21 46



_My application:_
screen shot 2017-07-06 at 19 06 17

_Customized text with extra colons:_
screen shot 2017-07-06 at 19 06 55

_Customized text without extra colons:_
screen shot 2017-07-06 at 19 18 52

File: app/views/chapters/_form.html.erb [hunk 1]
```diff
@@ -19,6 +19,7 @@
   <%= form.input :extra_question_1 %>
   <%= form.input :extra_question_2 %>
   <%= form.input :extra_question_3 %>
+  <%= form.input :default_email_text, label: "Customized Email Response", placeholder: "This will appear as the introduction in emails sent to applicants to your chapter." %>
```
Comment (jcn) [inline app/views/chapters/_form.html.erb]: I feel like the placeholder should be the actual default text, and then maybe we can include the helper text below the box. That way it'll indicate that even if this is not customized for the chapter that a message will be included by default.

In addition, the actual label (and helper) should be in the localization file, not hard-coded into the view; we shouldn't include any language-specific content in any views.

**Comment by arsduo** [comment]
Thanks for the feedback! I've localized the form fields and set it up so that if the chapter doesn't have (or deletes) the default text, it'll use the default value. This could result in a small amount of confusion if a chapter were to delete the text entirely, thinking that would remove the whole paragraph from the email -- if that (user-unfriendly situation) is worth addressing we could add a note to the form saying that if empty it will default back to the original value.

screen shot 2017-07-09 at 15 16 57

If the field is blank, applicants get the default text:
screen shot 2017-07-09 at 15 19 07

**Comment by jcn** [comment]
@arsduo I made a few changes to your PR: 

* renamed the column in the database
* made better use of the `simple_form` conventions
* added tests

Thanks for the PR!

**Comment by jcn** [comment]
@arsduo Oh also changed the form so the placeholder text is the default, but if you actually type in your own text, it overrides it. I hope this ends up being not too confusing for people. I guess we'll see!

**Comment by arsduo** [comment]
Thanks for getting this in! All those changes make sense. My thinking in making it editable was to allow people to change it (which is harder to do if it's a placeholder), but I'm guessing people will be fine either way.

While doing this, I noticed that the emails sent out are plain text, which looks a little old-school. Would you be interested in a PR making them HTML emails? We could allow chapters to enter either raw HTML or Markdown, since we already have the Redcarpet library for processing that.

**Comment by jcn** [comment]
@arsduo Feel free to submit a PR for an HTML version of the email, though there are really probably better things to do with your time. :smile:

That said, I don't think that we should encourage people to include raw HTML in their intro text - this is just going to lead to a Myspace-like situation where people are going to try to embed all sorts of things into these emails, many of which are just going to break the layout.

**Pull request merged**